src/__tests__/DeckEdit.test.js

```diff
@@ -1,20 +1,58 @@
-import { render, screen } from '@testing-library/react'
-import DeckEdit from '../pages/DeckEdit'
-import { BrowserRouter } from 'react-router-dom'
-import mockDecks from '../mockDecks'
+import { render, screen } from "@testing-library/react";
+import DeckEdit from "../pages/DeckEdit";
+import mockDecks from "../mockDecks";
+import { MemoryRouter, Routes, Route } from "react-router-dom";
+import mockUsers from "../mockUsers";
 
 describe("<DeckEdit />", () => {
-  beforeEach(() => {
+  const updateDeck = () => {
+    return
+  }
+  
```

Comment:
I don't understand lines 8-10.  Should those be there?

Reply:
Hi Trish, I pair programmed a little with Bao and that came from our App. Js app 
        <Route path={`/mydecks/:deck_id/edit`} element={<DeckEdit decks={decks} currentUser={currentUser} updateDeck={updateDeck} />} />

Reply:
DeckEdit component takes in an updateDeck prop and also has a handleSubmit function that calls the updateDeck function .  Without declaring updateDeck, the test return updateDeck as undefined which result in test failure. Since we're just doing simple on screen test, updateDeck is defined as shown to prevent test failure